modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -244,6 +243,7 @@ public void processOpts() {
             // the "okhttp-gson" library template requires "ApiCallback.mustache" for async call
             supportingFiles.add(new SupportingFile("ApiCallback.mustache", invokerFolder, "ApiCallback.java"));
             supportingFiles.add(new SupportingFile("ApiResponse.mustache", invokerFolder, "ApiResponse.java"));
+            supportingFiles.add(new SupportingFile("JSON.mustache", invokerFolder, "JSON.java"));
```

Comment:
Looking at [here in ApiClient](https://github.com/evigeant/swagger-codegen/blob/feature/simple-jaxrs-serialization/modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/ApiClient.mustache#L551) and [JSON](https://github.com/evigeant/swagger-codegen/blob/feature/simple-jaxrs-serialization/modules/swagger-codegen/src/main/resources/Java/libraries/jersey2/JSON.mustache), obviously the `JSON` class still exists in the jersey2 template, right?